x-pack/plugins/ml/public/ui_actions/index.ts

```diff
@@ -67,9 +68,9 @@ export function registerMlUiActions(
   uiActions.registerAction(addAnomalyChartsPanelAction);
 
   // Assign triggers
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addSingleMetricViewerPanelAction);
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addSwimlanePanelAction);
-  uiActions.addTriggerAction('ADD_PANEL_TRIGGER', addAnomalyChartsPanelAction);
+  uiActions.addTriggerAction(ADD_PANEL_TRIGGER, addSingleMetricViewerPanelAction);
```

Comment:
As part of these changes, could we change the section heading for the 'Machine Learning' section to 'Machine Learning and Analytics', as suggested by @arisonl and @teresaalvarezsoler ?

Reply:
Sure... I just took a look at the issue you linked.